Package.swift

```diff
@@ -27,7 +27,13 @@ let package = Package(
     .target(name: "_NumericsShims", dependencies: []),
     .target(name: "RealModule", dependencies: ["_NumericsShims"]),
     
-    .testTarget(name: "ComplexTests", dependencies: ["ComplexModule", "_NumericsShims"]),
+    .testTarget(
+      name: "ComplexTests",
+      dependencies: ["ComplexModule", "_NumericsShims"],
+      swiftSettings: [
+        .unsafeFlags(["-Xfrontend", "-enable-experimental-forward-mode-differentiation"])
```

Comment:
What compiler frontends support this, and what are the implications for projects that include Swift Numerics if we add an unsafe setting?

Reply:
Can you rebase this and put guards around anything that needs upstream swift changes to require swift > 5.3?